sample/get-bucket.c

```diff
@@ -15,16 +16,14 @@ int main(void) {
     uint64_t i;
 
     // Setup client credentials and then the actual client itself.
-    ds3_creds * creds = ds3_create_creds("cnlhbg==","ZIjGDQAs");
-    ds3_client * client = ds3_create_client("http://192.168.56.101:8080", creds);
+    ds3_creds * creds = ds3_create_creds(DS3_ACCESS_KEY, DS3_SECRET_KEY);
```

Comment:
Fair enough – should we include a sample shell script and bat file to set them? I agree buildFromEnv() is cleaner, I just want to facilitate a quick build and “hello world.”

From: Ryan Moore [mailto:notifications@github.com]
Sent: Tuesday, December 08, 2015 11:12 PM
To: SpectraLogic/ds3_c_sdk <ds3_c_sdk@noreply.github.com>
Cc: John Kranz <johnk@spectralogic.com>
Subject: Re: [ds3_c_sdk] Build in Debian 8 (CSDK-76) (#83)


In sample/get-bucket.c<https://github.com/SpectraLogic/ds3_c_sdk/pull/83#discussion_r47054432>:

> @@ -15,16 +16,14 @@ int main(void) {

>      uint64_t i;

>

>      // Setup client credentials and then the actual client itself.

> -    ds3_creds * creds = ds3_create_creds("cnlhbg==","ZIjGDQAs");

> -    ds3_client * client = ds3_create_client("http://192.168.56.101:8080", creds);

> +    ds3_creds * creds = ds3_create_creds(DS3_ACCESS_KEY, DS3_SECRET_KEY);

Instead of having this all defined in a header file, just use the ds3_create_client_from_env call instead.

—
Reply to this email directly or view it on GitHub<https://github.com/SpectraLogic/ds3_c_sdk/pull/83/files#r47054432>.